fulltest.sh

```diff
@@ -2,6 +2,7 @@ set -e # Stop the script on any failure or error
 docker-compose -f docker-compose.yml -f docker-compose.e2e.yml down
 docker-compose -f docker-compose.yml -f docker-compose.e2e.yml build
 docker-compose run backend lein test
+docker-compose run frontend yarn test
```

Comment:
This is for Travis?

Reply:
No, I'm working on travis, this is for the local "fulltest".